src/renderer/windows/settings/Settings.vue

```diff
@@ -396,10 +407,23 @@ window.ytmd.handleUpdateDownloaded(() => {
               <p class="description">
                 User is Authenticated:
                 <span v-if="lastFMSessionKey" style="color: #4caf50">Yes</span>
+                <button v-if="lastFMSessionKey" class="lastfm-logout-button" @click="logoutLastFM">Logout</button>
```

Comment:
![image](https://github.com/ytmdesktop/ytmdesktop/assets/17199186/3d2e19e6-9f4b-43c7-a502-2f9d278f5774)

If possible could you have this to look like the Discord rich presences connection failed.
```html
          <div v-if="discordPresenceEnabled && discordPresenceConnectionFailed" class="setting indented">
            <p class="discord-failure">Discord connection could not be established after 30 attempts</p>
            <button @click="restartDiscordPresence">Retry</button>
          </div>
```
I suspect you just need to put the button outside of the `<p>` tag